src/web-auth/redirect.js

```diff
@@ -2,6 +2,7 @@ var CrossOriginAuthentication = require('./cross-origin-authentication');
 var Warn = require('../helper/warn');
 
 function Redirect(auth0, options) {
+  this.webAuth = auth0;
   this.baseOptions = options;
   this.client = auth0.client;
```

Comment:
could this line be removed now, since `this.webAuth.client` is the same as `this.client`?

Reply:
absolutely. I removed in the next commit